main.go

```diff
@@ -19,6 +17,11 @@ import (
 // Version is the steward version (set during build)
 var Version = "unreleased"
 
+const (
+	DefaultArgoCDImage      = "quay.io/argoproj/argocd:v2.0.4@sha256:976dfbfadb817ba59f4f641597a13df7b967cd5a1059c966fa843869c9463348"
+	DefaultArgoCDRedisImage = "docker.io/redis:6.2.4@sha256:f631ff6c898339306ffdb8369add5c12303ec3946610ef8d6f1d05f575942f0c"
```

Comment:
While you're at it, could you try using the Bitnami image on quay.io?
It's an open issue we have anyway (https://ticket.vshn.net/browse/SYN-288)

Reply:
Tracked in #80 .